fountain-core/src/main/java/android/arch/paging/PagingRequestHelper.java

```diff
@@ -15,17 +15,20 @@
  */
 package android.arch.paging;
 
-import android.support.annotation.AnyThread;
-import android.support.annotation.GuardedBy;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.annotation.VisibleForTesting;
 
 import java.util.Arrays;
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.Executor;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import androidx.annotation.AnyThread;
+import androidx.annotation.GuardedBy;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import androidx.paging.DataSource;
```

Comment:
can you check? this wasn't a previous import

Reply:
we should update the detekt version, the new version checks it!